stubs/torch/__init__.pyi

```diff
@@ -28,6 +28,7 @@ from . import cuda as cuda
 from . import optim as optim
 from . import nn as nn
 from . import testing as testing
+from . import utils as utils
```

Comment:
these two pyi changes are to make mypy testing happy. Don't know why it doesn't show up in earlier testing.

Reply:
lgtm!